### ISSUE TITLE
`jsx-indent`: issue with multiline function calls

### ISSUE THREAD

**Issue by ljharb** [yannickcr/eslint-plugin-react#895]
I have this code:
```jsx
    it('passes', () => assertPasses(
      foo('bar'),
      (
        <div>
          <span />
          <span />
          <span />
          <span />
        </div>
      ),
      'baz',
    ));
```

With these settings:
```sh
'react/jsx-indent': ['error', 2],
```

I get this warning: `Expected indentation of 6 space characters but found 8`

This code passes:
```jsx
    it('passes', () => assertPasses(
      foo('bar'),
      (<div>
        <span />
        <span />
        <span />
        <span />
      </div>),
      'baz',
    ));
```

I don't mind if the latter example passes - but the former example should be correct also.

**Comment by ljharb** [comment]
Thanks!
